textile.go

```diff
@@ -55,6 +62,10 @@ type CafeOptions struct {
 	DBTLS       bool   `long:"cafe-db-tls" description:"use TLS for the cafe mongo db connection"`
 	TokenSecret string `long:"cafe-token-secret" description:"set the cafe token secret"`
 	ReferralKey string `long:"cafe-referral-key" description:"set the cafe referral key"`
+
+	// Slack hook url settings
```

Comment:
Added these to CafeOptions because then it'll work nicely in both shell and daemon modes. _Could_ do this as its own top-level group, but seems to fit here conceptually.